csprng.php

```diff
@@ -25,13 +24,35 @@ function random_bytes($bytesLength)
     }
 
     // @todo Validate a max $bytesLength value here?
+    
+    /**
+     * If we can safely use mcrypt, let's use mcrypt
+     */
+    if (function_exists('mcrypt_create_iv') && version_compare(PHP_VERSION, '5.3.7') >= 0) {
```

Comment:
The RFC talks about possibly [deprecating `mcrypt_create_iv()`](https://wiki.php.net/rfc/easy_userland_csprng#future_scope). And the ultimate reason for posting this repo was for a proof-of-concept, not to provided a CSPRNG for people to use in real-life projects. (I should probably put a big warning in the README lol).